packages/jest-jasmine2/src/jasmine/Spec.js

```diff
@@ -110,24 +110,29 @@ Spec.prototype.execute = function(onComplete, enabled) {
   }
 };
 
-Spec.prototype.onException = function onException(e) {
-  if (Spec.isPendingSpecException(e)) {
-    this.pend(extractCustomPendingMessage(e));
+Spec.prototype.onException = function onException(error) {
+  if (Spec.isPendingSpecException(error)) {
+    this.pend(extractCustomPendingMessage(error));
     return;
   }
 
-  if (e instanceof ExpectationFailed) {
+  if (error instanceof ExpectationFailed) {
     return;
   }
 
+  if (error instanceof require('assert').AssertionError) {
+    const assertionErrorMessage = require('jest-matchers/build/assert-support');
```

Comment:
We shouldn't do this. Can we move assert-support out of jest-matchers? Can it be in jest-jasmine2 instead?